testing/login/Login003 - login as team.md

```diff
@@ -6,14 +6,13 @@
 
 *Preconditions*
 * Login page is opened.
-* The team has already started the game.
-* There are some cyphers created by admin.
+* The team is created.
 
 |Test Step|Expected Result|
 |---------|---------------|
 |Enter team name to the username field.|Username is entered.|
 |Enter valid PIN to the password field.|PIN is entered.|
-|Click on Login button.|Team name is displayed in the header.<br>Team score is displayed in the header.<br>The list of cyphers is displayed.|
+|Click on Login button.|Team name is displayed in the header.
```

Comment:
chybi | na konci
+ nechal bych tam: The list of cyphers is displayed.

Reply:
Neni - list of cypher displayed, protože tým je nově vytvořený a ještě nezahájit hru, tak je tam jen tlačítko zahájit hru

Reply:
tak pridat podminku, ze menu List Sifer bude zvyraznene

Reply:
nerozumím. :-( Když vyrobím nový tým a ten se přihlásí, tak stránka vypadá takto 
![Capture](https://user-images.githubusercontent.com/42601181/58481184-28068100-815c-11e9-8e66-7f9377a415e5.JPG)

Reply:
joa aha, tady neni ta hlavicka jako v organizatorovi/adminovi, tady je to vsechno v burger-menu schovane :)
tak to nech byt jak je ;)